go/cmd/vtctldclient/internal/command/tablets.go

```diff
@@ -96,6 +96,30 @@ Valid output formats are "awk" and "json".`,
 		Args:                  cobra.ExactArgs(1),
 		RunE:                  commandRefreshStateByShard,
 	}
+	// SetWritable makes a SetWritable gRPC call to a vtctld.
+	SetWritable = &cobra.Command{
+		Use:                   "SetWritable [--read-only] <alias>",
```

Comment:
This feels less intuitive from a UX perspective than `SetWritable <true/false>`

Reply:
fixed!